[TT-16075] Pump 1.13.1 RNs.mdx (#888)
TykTechnologies/tyk-docs: developer-support/release-notes/pump.mdx

## 1.13 Release Notes

### 1.13.1 Release Notes

> #### Release Date xx November 2025

Reviewer: ```suggestion
#### Release Date xx November 2025
```
TODO: Change this

#### Release Highlights

In this patch release, we fixed high-priority CVEs. For a comprehensive list of changes, please refer to the detailed [changelog](#Changelog-v1.13.1).

#### Breaking Changes

#### Dependencies

##### 3rd Party Dependencies & Tools

| Third Party Dependency                                    | Tested Versions   | Compatible Versions      |
| :--------------------------------------------------------- | :----------------- | :------------------------ | 
| [MongoDB](https://www.mongodb.com/try/download/community) | 5.x, 6.x, and 7.0 | 4.4.x, 5.x, 6.x, and 7.0 |
| [PostgreSQL](https://www.postgresql.org/download/)        | 13.x - 17.x       | 13.x - 17.x              | 
| [Redis](https://redis.io/download/)                       | 6.x - 7.0         | 6.x - 7.x                | 
| [Valkey](https://valkey.io/download/)                     | 8.0.x, 8.1.x      | 7.2.x, 8.0.x, 8.1.x      |                            

Given the time difference between your upgrade and the release of this version, we recommend customers verify the ongoing support of third-party dependencies they install, as their status may have changed since the release.

#### Deprecations

> Deprecated the global configuration options [DecodeRawRequest](/tyk-pump/tyk-pump-configuration/tyk-pump-environment-variables#raw_request_decoded) and [DecodeRawResponse](/tyk-pump/tyk-pump-configuration/tyk-pump-environment-variables#raw_response_decoded).

Reviewer: ```suggestion
There are no deprecations in this release.
```

For further information, please check the [changelog](#Changelog-v1.13.1) below.

#### Upgrade instructions

> For users currently on v1.12.2, we strongly recommend promptly upgrading to the latest release. If you are working with an older version (lower major), it is advisable to bypass version 1.12.2 and proceed directly to this latest patch release.

Reviewer: ```suggestion
For users currently on v1.13.0, we strongly recommend promptly upgrading to the latest release. If you are working with an older version (lower major), it is advisable to bypass version 1.13.0 and proceed directly to this latest patch release.
```

#### Deprecations

Deprecated the global configuration options [DecodeRawRequest](/tyk-pump/tyk-pump-configuration/tyk-pump-environment-variables#raw_request_decoded) and [DecodeRawResponse](/tyk-pump/tyk-pump-configuration/tyk-pump-environment-variables#raw_response_decoded).
For further information, please check the [changelog](#Changelog-v1.13.1) below.

#### Upgrade instructions

For users currently on v1.12.2, we strongly recommend promptly upgrading to the latest release. If you are working with an older version (lower major), it is advisable to bypass version 1.12.2 and proceed directly to this latest patch release.
<br/>
Go to the [Upgrading Tyk](#upgrading-tyk) section for detailed upgrade Instructions.

#### Downloads
- [Docker Image v1.13.1](https://hub.docker.com/r/tykio/tyk-pump-docker-pub/tags?page=&page_size=&ordering=&name=v1.13.1)
  - ```bash
    docker pull tykio/tyk-pump-docker-pub:v1.13.1
    ```
- Source code tarball for OSS - [GH Tyk Pump Repo](https://github.com/TykTechnologies/tyk-pump/releases/tag/v1.13.1)

#### Changelog
<a id="Changelog-v1.13.1"></a>

> 

Reviewer: Changelog is missing
